contrib/forms/evaluation/FormEvaluation.class.php

```diff
@@ -63,8 +63,8 @@ function populate()
     {
         parent::populate();
 
-        $sql = "SELECT name from form_evaluation_checks where foreign_id = '" . add_escape_custom($this->id) . "'";
-        $results = sqlQ($sql);
+        $sql = "SELECT name from form_evaluation_checks where foreign_id = ?";
+        $results = sqlQ($sql, array(add_escape_custom($this->id)));
```

Comment:
Remove the add_escape_custom() wrapper.
(note this is a function that will sql escape variables; but binding is better)